[20230614T134247.758] [DEBUG] [SB_just_keep_picking] -- Failed Service:= picker.1
[20230614T134247.758] [DEBUG] [SB_just_keep_picking] -- Attempting to Disable Robot 1
[20230614T134247.758] [DEBUG] [SB_just_keep_picking] -- Checking connection to rcoop
[20230614T134247.759] [DEBUG] [SB_just_keep_picking] -- Checking connection to autodrive
[20230614T134247.759] [DEBUG] [SB_just_keep_picking] -- Connected to all master level services
[20230614T134247.768] [INFO] [servact     ] -- Service Disable Operation - Starting
[20230614T134247.769] [DEBUG] [servact     ] -- Disable Batch: {'rcoop.0', 'autodrive.0'}
[20230614T134247.769] [INFO] [servact     ] -- rcoop.0 not in list of services to disable.  Skipping
[20230614T134247.769] [INFO] [servact     ] -- autodrive.0 not in list of services to disable.  Skipping
[20230614T134247.770] [DEBUG] [servact     ] -- Disable Batch: {'picker.1'}
[20230614T134247.771] [DEBUG] [servact     ] -- Disable Batch: {'tracker.1', 'gripper.1'}
[20230614T134249.153] [DEBUG] [servact     ] -- Disable Batch: {'obscore.1', 'robot.1'}
[20230614T134249.481] [DEBUG] [SB_just_keep_picking] -- Reached min number of robots allowed to operate
[31m[20230614T134249.481] [ERROR] [harv.beh.fsm] -- FAULT: {'ts': 1686775369.4488559, 'value': "Twist ZMQ Error:\nInvokation: InvokationTuple(servName='robot.4', cmd='move', args=([(Pos: [ 325.    -489.623 -650.1      0.   ], Spd: 2000.000, RotSpd: 205.714, Tol: 20.000, sysid: None, uuid: D29LnWwBsdRm2cgfBYpTjz, ts: 1686775369.826138, simultaneous: True, strict: True), (Pos: [ 170.    -489.623 -600.     -10.   ], Spd: 1000.000, RotSpd: 102.857, Tol: 0.100, sysid: None, uuid: bxeU86R4mSyqNhmcbcA7t3, ts: 1686775370.1318908, simultaneous: True, strict: True), (Pos: [ 430.    -489.623 -700.       0.   ], Spd: 1000.000, RotSpd: 102.857, Tol: 0.100, sysid: None, uuid: JfMsoTvyL8b5GRSnV4eufX, ts: 1686775370.5533156, simultaneous: True, strict: True), (Pos: [ 550.    -489.623 -750.      10.   ], Spd: 1000.000, RotSpd: 102.857, Tol: 0.100, sysid: None, uuid: TSU3bpg5xJyfyCH4spWEU6, ts: 1686775370.8258698, simultaneous: True, strict: True)],), kwargs={'clear': True}, ts=1686775369.4172866)\nType: <class 'aft_tbot.exceptions.RangeOfMotionException.RangeOfMotionException'>\nMessage: \ntray/grip_org violations: \nWaypt w/ uuid: D29LnWwBsdRm2cgfBYpTjz and pos: [ 325.    -489.623 -650.1      0.   ]\n[-118.15   657.558 -691.572]\n[-116.873  639.224 -705.096]\n[-115.909  625.39  -715.3  ]\n[-115.259  616.055 -722.186]\n[-114.922  611.22  -725.752]\nWaypt w/ uuid: bxeU86R4mSyqNhmcbcA7t3 and pos: [ 170.    -489.623 -600.      -0.175]\n[-115.111  610.477 -726.222]\n[-117.577  610.477 -725.425]\n[-122.707  610.477 -723.767]\n[-130.501  610.477 -721.248]\n[-140.96   610.477 -717.867]\n[-154.083  610.477 -713.625]\n[-169.871  610.477 -708.522]\n[-188.216  610.477 -702.593]\n[-207.142  610.477 -696.475]\n[-224.092  610.477 -690.996]\n[-238.379  610.477 -686.379]\n[-250.001  610.477 -682.622]\n[-258.958  610.477 -679.727]\n[-265.251  610.477 -677.693]\n[-268.88   610.477 -676.52 ]\nWaypt w/ uuid: JfMsoTvyL8b5GRSnV4eufX and pos: [ 430.    -489.623 -700.       0.   ]\n[-269.845  610.477 -676.21 ]\n[-268.178  610.477 -676.851]\n[-263.897  610.477 -678.497]\n[-257.003  610.477 -681.149]\n[-247.495  610.477 -684.806]\n[-235.374  610.477 -689.468]\n[-220.64   610.477 -695.135]\n[-203.292  610.477 -701.807]\n[-184.625  610.477 -708.987]\n[-165.958  610.477 -716.166]\n[-147.29   610.477 -723.346]\n[-128.623  610.477 -730.526]\n[-109.956  610.477 -737.705]\n[ -91.289  610.477 -744.885]\n[ -72.679  610.477 -752.043]\n[ -55.867  610.477 -758.509]\n[ -41.668  610.477 -763.97 ]\n[ -30.083  610.477 -768.426]\n[ -21.111  610.477 -771.876]\n[ -14.753  610.477 -774.322]\n[ -11.008  610.477 -775.762]\nWaypt w/ uuid: TSU3bpg5xJyfyCH4spWEU6 and pos: [ 550.    -489.623 -750.       0.175]\n[  -9.864  610.477 -776.202]\n[  -8.399  610.477 -776.813]\n[  -4.349  610.477 -778.501]\n[   2.286  610.477 -781.265]\n[  11.506  610.477 -785.107]\n[  23.31   610.477 -790.025]\n[  37.699  610.477 -796.02 ]\n[  54.506  610.477 -803.023]\n[  70.171  610.477 -809.55 ]\n[  83.251  610.477 -815.   ]\n[  93.746  610.477 -819.373]\n[ 101.657  610.477 -822.669]\n[ 106.983  610.477 -824.889]\n[ 109.724  610.477 -826.031]\n[ 110.13   610.477 -826.2  ]\n\nServer Side Traceback: Traceback: <class 'aft_tbot.exceptions.RangeOfMotionException.RangeOfMotionException'>: \ntray/grip_org violations: \nWaypt w/ uuid: D29LnWwBsdRm2cgfBYpTjz and pos: [ 325.    -489.623 -650.1      0.   ]\n[-118.15   657.558 -691.572]\n[-116.873  639.224 -705.096]\n[-115.909  625.39  -715.3  ]\n[-115.259  616.055 -722.186]\n[-114.922  611.22  -725.752]\nWaypt w/ uuid: bxeU86R4mSyqNhmcbcA7t3 and pos: [ 170.    -489.623 -600.      -0.175]\n[-115.111  610.477 -726.222]\n[-117.577  610.477 -725.425]\n[-122.707  610.477 -723.767]\n[-130.501  610.477 -721.248]\n[-140.96   610.477 -717.867]\n[-154.083  610.477 -713.625]\n[-169.871  610.477 -708.522]\n[-188.216  610.477 -702.593]\n[-207.142  610.477 -696.475]\n[-224.092  610.477 -690.996]\n[-238.379  610.477 -686.379]\n[-250.001  610.477 -682.622]\n[-258.958  610.477 -679.727]\n[-265.251  610.477 -677.693]\n[-268.88   610.477 -676.52 ]\nWaypt w/ uuid: JfMsoTvyL8b5GRSnV4eufX and pos: [ 430.    -489.623 -700.       0.   ]\n[-269.845  610.477 -676.21 ]\n[-268.178  610.477 -676.851]\n[-263.897  610.477 -678.497]\n[-257.003  610.477 -681.149]\n[-247.495  610.477 -684.806]\n[-235.374  610.477 -689.468]\n[-220.64   610.477 -695.135]\n[-203.292  610.477 -701.807]\n[-184.625  610.477 -708.987]\n[-165.958  610.477 -716.166]\n[-147.29   610.477 -723.346]\n[-128.623  610.477 -730.526]\n[-109.956  610.477 -737.705]\n[ -91.289  610.477 -744.885]\n[ -72.679  610.477 -752.043]\n[ -55.867  610.477 -758.509]\n[ -41.668  610.477 -763.97 ]\n[ -30.083  610.477 -768.426]\n[ -21.111  610.477 -771.876]\n[ -14.753  610.477 -774.322]\n[ -11.008  610.477 -775.762]\nWaypt w/ uuid: TSU3bpg5xJyfyCH4spWEU6 and pos: [ 550.    -489.623 -750.       0.175]\n[  -9.864  610.477 -776.202]\n[  -8.399  610.477 -776.813]\n[  -4.349  610.477 -778.501]\n[   2.286  610.477 -781.265]\n[  11.506  610.477 -785.107]\n[  23.31   610.477 -790.025]\n[  37.699  610.477 -796.02 ]\n[  54.506  610.477 -803.023]\n[  70.171  610.477 -809.55 ]\n[  83.251  610.477 -815.   ]\n[  93.746  610.477 -819.373]\n[ 101.657  610.477 -822.669]\n[ 106.983  610.477 -824.889]\n[ 109.724  610.477 -826.031]\n[ 110.13   610.477 -826.2  ]\n\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:517:errback\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:580:_startRunCallbacks\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:662:_runCallbacks\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:1514:gotResult\n--- <exception caught here> ---\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:1443:_inlineCallbacks\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/python/failure.py:500:throwExceptionIntoGenerator\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robotbehavior.py:365:move\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:1445:_inlineCallbacks\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robotbehavior.py:409:_move\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robot.py:216:move\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/motionctl.py:482:move\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/motionctl.py:511:_expose_path\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robotbehavior.py:947:validate_path\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/defaultRobotVolumes.py:95:check\n", 'traceback': 'Traceback (most recent call last):\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 1514, in gotResult\n    current_context.run(_inlineCallbacks, r, g, status)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 1504, in _inlineCallbacks\n    status.deferred.errback()\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 517, in errback\n    self._startRunCallbacks(fail)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 580, in _startRunCallbacks\n    self._runCallbacks()\n--- <exception caught here> ---\n  File "/home/aft/_harv-aft-py-packages/aft-core/src/aft_core/asm/AsyncStateMachine.py", line 985, in _process_curr_state\n    result = yield self._stateTransDefer\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 662, in _runCallbacks\n    current.result = callback(current.result, *args, **kw)\n  File "/home/aft/_harv-aft-py-packages/aft-core/src/aft_core/twisted/twistutils.py", line 179, in trap_cancel\n    failType = fail.trap(defer.CancelledError, DeferredListError)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/python/failure.py", line 450, in trap\n    self.raiseException()\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/python/failure.py", line 475, in raiseException\n    raise self.value.with_traceback(self.tb)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 167, in maybeDeferred\n    result = f(*args, **kw)\n  File "/home/aft/_harv-aft-py-packages/aft-picker/src/aft_picker/pickerbehaviorbase.py", line 306, in state_handle_abort\n    raise child.finished_result.data\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 662, in _runCallbacks\n    current.result = callback(current.result, *args, **kw)\n  File "/home/aft/_harv-aft-py-packages/aft-core/src/aft_core/twisted/twistutils.py", line 179, in trap_cancel\n    failType = fail.trap(defer.CancelledError, DeferredListError)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/python/failure.py", line 450, in trap\n    self.raiseException()\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/python/failure.py", line 475, in raiseException\n    raise self.value.with_traceback(self.tb)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 1443, in _inlineCallbacks\n    result = current_context.run(result.throwExceptionIntoGenerator, g)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/python/failure.py", line 500, in throwExceptionIntoGenerator\n    return g.throw(self.type, self.value, self.tb)\n  File "/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robotclient.py", line 437, in track\n    ret = yield self._track(*args, **kwargs)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 1443, in _inlineCallbacks\n    result = current_context.run(result.throwExceptionIntoGenerator, g)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/python/failure.py", line 500, in throwExceptionIntoGenerator\n    return g.throw(self.type, self.value, self.tb)\n  File "/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robotclient.py", line 443, in _track\n    ret = yield self._try_motion(self.move_rpc, *args, **kwargs)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 1443, in _inlineCallbacks\n    result = current_context.run(result.throwExceptionIntoGenerator, g)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/python/failure.py", line 500, in throwExceptionIntoGenerator\n    return g.throw(self.type, self.value, self.tb)\n  File "/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robotclient.py", line 375, in _try_motion\n    ret = yield self.cur_try_motion_d\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 1443, in _inlineCallbacks\n    result = current_context.run(result.throwExceptionIntoGenerator, g)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/python/failure.py", line 500, in throwExceptionIntoGenerator\n    return g.throw(self.type, self.value, self.tb)\n  File "/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robotclient.py", line 384, in _try_motion_worker\n    ret = yield self.active_motion_rpc_d\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 662, in _runCallbacks\n    current.result = callback(current.result, *args, **kw)\n  File "/home/aft/_harv-aft-py-packages/aft-core/src/aft_core/zmq/ZmqClient.py", line 353, in _handle_resp\n    raise TwistZmqException(resp)\naft_core.tools.TwistZmqException.TwistZmqException: Twist ZMQ Error:\nInvokation: InvokationTuple(servName=\'robot.4\', cmd=\'move\', args=([(Pos: [ 325.    -489.623 -650.1      0.   ], Spd: 2000.000, RotSpd: 205.714, Tol: 20.000, sysid: None, uuid: D29LnWwBsdRm2cgfBYpTjz, ts: 1686775369.826138, simultaneous: True, strict: True), (Pos: [ 170.    -489.623 -600.     -10.   ], Spd: 1000.000, RotSpd: 102.857, Tol: 0.100, sysid: None, uuid: bxeU86R4mSyqNhmcbcA7t3, ts: 1686775370.1318908, simultaneous: True, strict: True), (Pos: [ 430.    -489.623 -700.       0.   ], Spd: 1000.000, RotSpd: 102.857, Tol: 0.100, sysid: None, uuid: JfMsoTvyL8b5GRSnV4eufX, ts: 1686775370.5533156, simultaneous: True, strict: True), (Pos: [ 550.    -489.623 -750.      10.   ], Spd: 1000.000, RotSpd: 102.857, Tol: 0.100, sysid: None, uuid: TSU3bpg5xJyfyCH4spWEU6, ts: 1686775370.8258698, simultaneous: True, strict: True)],), kwargs={\'clear\': True}, ts=1686775369.4172866)\nType: <class \'aft_tbot.exceptions.RangeOfMotionException.RangeOfMotionException\'>\nMessage: \ntray/grip_org violations: \nWaypt w/ uuid: D29LnWwBsdRm2cgfBYpTjz and pos: [ 325.    -489.623 -650.1      0.   ]\n[-118.15   657.558 -691.572]\n[-116.873  639.224 -705.096]\n[-115.909  625.39  -715.3  ]\n[-115.259  616.055 -722.186]\n[-114.922  611.22  -725.752]\nWaypt w/ uuid: bxeU86R4mSyqNhmcbcA7t3 and pos: [ 170.    -489.623 -600.      -0.175]\n[-115.111  610.477 -726.222]\n[-117.577  610.477 -725.425]\n[-122.707  610.477 -723.767]\n[-130.501  610.477 -721.248]\n[-140.96   610.477 -717.867]\n[-154.083  610.477 -713.625]\n[-169.871  610.477 -708.522]\n[-188.216  610.477 -702.593]\n[-207.142  610.477 -696.475]\n[-224.092  610.477 -690.996]\n[-238.379  610.477 -686.379]\n[-250.001  610.477 -682.622]\n[-258.958  610.477 -679.727]\n[-265.251  610.477 -677.693]\n[-268.88   610.477 -676.52 ]\nWaypt w/ uuid: JfMsoTvyL8b5GRSnV4eufX and pos: [ 430.    -489.623 -700.       0.   ]\n[-269.845  610.477 -676.21 ]\n[-268.178  610.477 -676.851]\n[-263.897  610.477 -678.497]\n[-257.003  610.477 -681.149]\n[-247.495  610.477 -684.806]\n[-235.374  610.477 -689.468]\n[-220.64   610.477 -695.135]\n[-203.292  610.477 -701.807]\n[-184.625  610.477 -708.987]\n[-165.958  610.477 -716.166]\n[-147.29   610.477 -723.346]\n[-128.623  610.477 -730.526]\n[-109.956  610.477 -737.705]\n[ -91.289  610.477 -744.885]\n[ -72.679  610.477 -752.043]\n[ -55.867  610.477 -758.509]\n[ -41.668  610.477 -763.97 ]\n[ -30.083  610.477 -768.426]\n[ -21.111  610.477 -771.876]\n[ -14.753  610.477 -774.322]\n[ -11.008  610.477 -775.762]\nWaypt w/ uuid: TSU3bpg5xJyfyCH4spWEU6 and pos: [ 550.    -489.623 -750.       0.175]\n[  -9.864  610.477 -776.202]\n[  -8.399  610.477 -776.813]\n[  -4.349  610.477 -778.501]\n[   2.286  610.477 -781.265]\n[  11.506  610.477 -785.107]\n[  23.31   610.477 -790.025]\n[  37.699  610.477 -796.02 ]\n[  54.506  610.477 -803.023]\n[  70.171  610.477 -809.55 ]\n[  83.251  610.477 -815.   ]\n[  93.746  610.477 -819.373]\n[ 101.657  610.477 -822.669]\n[ 106.983  610.477 -824.889]\n[ 109.724  610.477 -826.031]\n[ 110.13   610.477 -826.2  ]\n\nServer Side Traceback: Traceback: <class \'aft_tbot.exceptions.RangeOfMotionException.RangeOfMotionException\'>: \ntray/grip_org violations: \nWaypt w/ uuid: D29LnWwBsdRm2cgfBYpTjz and pos: [ 325.    -489.623 -650.1      0.   ]\n[-118.15   657.558 -691.572]\n[-116.873  639.224 -705.096]\n[-115.909  625.39  -715.3  ]\n[-115.259  616.055 -722.186]\n[-114.922  611.22  -725.752]\nWaypt w/ uuid: bxeU86R4mSyqNhmcbcA7t3 and pos: [ 170.    -489.623 -600.      -0.175]\n[-115.111  610.477 -726.222]\n[-117.577  610.477 -725.425]\n[-122.707  610.477 -723.767]\n[-130.501  610.477 -721.248]\n[-140.96   610.477 -717.867]\n[-154.083  610.477 -713.625]\n[-169.871  610.477 -708.522]\n[-188.216  610.477 -702.593]\n[-207.142  610.477 -696.475]\n[-224.092  610.477 -690.996]\n[-238.379  610.477 -686.379]\n[-250.001  610.477 -682.622]\n[-258.958  610.477 -679.727]\n[-265.251  610.477 -677.693]\n[-268.88   610.477 -676.52 ]\nWaypt w/ uuid: JfMsoTvyL8b5GRSnV4eufX and pos: [ 430.    -489.623 -700.       0.   ]\n[-269.845  610.477 -676.21 ]\n[-268.178  610.477 -676.851]\n[-263.897  610.477 -678.497]\n[-257.003  610.477 -681.149]\n[-247.495  610.477 -684.806]\n[-235.374  610.477 -689.468]\n[-220.64   610.477 -695.135]\n[-203.292  610.477 -701.807]\n[-184.625  610.477 -708.987]\n[-165.958  610.477 -716.166]\n[-147.29   610.477 -723.346]\n[-128.623  610.477 -730.526]\n[-109.956  610.477 -737.705]\n[ -91.289  610.477 -744.885]\n[ -72.679  610.477 -752.043]\n[ -55.867  610.477 -758.509]\n[ -41.668  610.477 -763.97 ]\n[ -30.083  610.477 -768.426]\n[ -21.111  610.477 -771.876]\n[ -14.753  610.477 -774.322]\n[ -11.008  610.477 -775.762]\nWaypt w/ uuid: TSU3bpg5xJyfyCH4spWEU6 and pos: [ 550.    -489.623 -750.       0.175]\n[  -9.864  610.477 -776.202]\n[  -8.399  610.477 -776.813]\n[  -4.349  610.477 -778.501]\n[   2.286  610.477 -781.265]\n[  11.506  610.477 -785.107]\n[  23.31   610.477 -790.025]\n[  37.699  610.477 -796.02 ]\n[  54.506  610.477 -803.023]\n[  70.171  610.477 -809.55 ]\n[  83.251  610.477 -815.   ]\n[  93.746  610.477 -819.373]\n[ 101.657  610.477 -822.669]\n[ 106.983  610.477 -824.889]\n[ 109.724  610.477 -826.031]\n[ 110.13   610.477 -826.2  ]\n\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:517:errback\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:580:_startRunCallbacks\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:662:_runCallbacks\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:1514:gotResult\n--- <exception caught here> ---\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:1443:_inlineCallbacks\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/python/failure.py:500:throwExceptionIntoGenerator\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robotbehavior.py:365:move\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:1445:_inlineCallbacks\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robotbehavior.py:409:_move\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robot.py:216:move\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/motionctl.py:482:move\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/motionctl.py:511:_expose_path\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robotbehavior.py:947:validate_path\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/defaultRobotVolumes.py:95:check\n\n', 'code': 14, 'aux_uuid': '069fa944-0af4-11ee-868f-3a5051573772', 'ep': 'picker.4'}[0m[27m
[36m[20230614T134249.482] [TRANSITION] [harv.beh.fsm] -- Exiting: state_picking[0m[27m
[20230614T134249.482] [DEBUG] [harv.beh.fsm] -- starting report generation
[20230614T134249.490] [INFO] [servact     ] -- Service Disable Operation - Starting
[20230614T134249.492] [DEBUG] [servact     ] -- Disable Batch: {'rcoop.0', 'fieldconditions.0', 'logrec.0', 'autodrive.0'}
[20230614T134249.494] [DEBUG] [harv.beh.fsm] -- Disabling Services in Progress - Exited Picking
[36m[20230614T134249.495] [TRANSITION] [harv.beh.fsm] -- Entering: state_error[0m[27m
[20230614T134249.511] [DEBUG] [SB_just_keep_picking] -- Reached min number of robots allowed to operate
[31m[20230614T134249.521] [ERROR] [harv.beh.fsm] -- FAULT: {'ts': 1686775369.4488559, 'value': "Twist ZMQ Error:\nInvokation: InvokationTuple(servName='robot.4', cmd='move', args=([(Pos: [ 325.    -489.623 -650.1      0.   ], Spd: 2000.000, RotSpd: 205.714, Tol: 20.000, sysid: None, uuid: D29LnWwBsdRm2cgfBYpTjz, ts: 1686775369.826138, simultaneous: True, strict: True), (Pos: [ 170.    -489.623 -600.     -10.   ], Spd: 1000.000, RotSpd: 102.857, Tol: 0.100, sysid: None, uuid: bxeU86R4mSyqNhmcbcA7t3, ts: 1686775370.1318908, simultaneous: True, strict: True), (Pos: [ 430.    -489.623 -700.       0.   ], Spd: 1000.000, RotSpd: 102.857, Tol: 0.100, sysid: None, uuid: JfMsoTvyL8b5GRSnV4eufX, ts: 1686775370.5533156, simultaneous: True, strict: True), (Pos: [ 550.    -489.623 -750.      10.   ], Spd: 1000.000, RotSpd: 102.857, Tol: 0.100, sysid: None, uuid: TSU3bpg5xJyfyCH4spWEU6, ts: 1686775370.8258698, simultaneous: True, strict: True)],), kwargs={'clear': True}, ts=1686775369.4172866)\nType: <class 'aft_tbot.exceptions.RangeOfMotionException.RangeOfMotionException'>\nMessage: \ntray/grip_org violations: \nWaypt w/ uuid: D29LnWwBsdRm2cgfBYpTjz and pos: [ 325.    -489.623 -650.1      0.   ]\n[-118.15   657.558 -691.572]\n[-116.873  639.224 -705.096]\n[-115.909  625.39  -715.3  ]\n[-115.259  616.055 -722.186]\n[-114.922  611.22  -725.752]\nWaypt w/ uuid: bxeU86R4mSyqNhmcbcA7t3 and pos: [ 170.    -489.623 -600.      -0.175]\n[-115.111  610.477 -726.222]\n[-117.577  610.477 -725.425]\n[-122.707  610.477 -723.767]\n[-130.501  610.477 -721.248]\n[-140.96   610.477 -717.867]\n[-154.083  610.477 -713.625]\n[-169.871  610.477 -708.522]\n[-188.216  610.477 -702.593]\n[-207.142  610.477 -696.475]\n[-224.092  610.477 -690.996]\n[-238.379  610.477 -686.379]\n[-250.001  610.477 -682.622]\n[-258.958  610.477 -679.727]\n[-265.251  610.477 -677.693]\n[-268.88   610.477 -676.52 ]\nWaypt w/ uuid: JfMsoTvyL8b5GRSnV4eufX and pos: [ 430.    -489.623 -700.       0.   ]\n[-269.845  610.477 -676.21 ]\n[-268.178  610.477 -676.851]\n[-263.897  610.477 -678.497]\n[-257.003  610.477 -681.149]\n[-247.495  610.477 -684.806]\n[-235.374  610.477 -689.468]\n[-220.64   610.477 -695.135]\n[-203.292  610.477 -701.807]\n[-184.625  610.477 -708.987]\n[-165.958  610.477 -716.166]\n[-147.29   610.477 -723.346]\n[-128.623  610.477 -730.526]\n[-109.956  610.477 -737.705]\n[ -91.289  610.477 -744.885]\n[ -72.679  610.477 -752.043]\n[ -55.867  610.477 -758.509]\n[ -41.668  610.477 -763.97 ]\n[ -30.083  610.477 -768.426]\n[ -21.111  610.477 -771.876]\n[ -14.753  610.477 -774.322]\n[ -11.008  610.477 -775.762]\nWaypt w/ uuid: TSU3bpg5xJyfyCH4spWEU6 and pos: [ 550.    -489.623 -750.       0.175]\n[  -9.864  610.477 -776.202]\n[  -8.399  610.477 -776.813]\n[  -4.349  610.477 -778.501]\n[   2.286  610.477 -781.265]\n[  11.506  610.477 -785.107]\n[  23.31   610.477 -790.025]\n[  37.699  610.477 -796.02 ]\n[  54.506  610.477 -803.023]\n[  70.171  610.477 -809.55 ]\n[  83.251  610.477 -815.   ]\n[  93.746  610.477 -819.373]\n[ 101.657  610.477 -822.669]\n[ 106.983  610.477 -824.889]\n[ 109.724  610.477 -826.031]\n[ 110.13   610.477 -826.2  ]\n\nServer Side Traceback: Traceback: <class 'aft_tbot.exceptions.RangeOfMotionException.RangeOfMotionException'>: \ntray/grip_org violations: \nWaypt w/ uuid: D29LnWwBsdRm2cgfBYpTjz and pos: [ 325.    -489.623 -650.1      0.   ]\n[-118.15   657.558 -691.572]\n[-116.873  639.224 -705.096]\n[-115.909  625.39  -715.3  ]\n[-115.259  616.055 -722.186]\n[-114.922  611.22  -725.752]\nWaypt w/ uuid: bxeU86R4mSyqNhmcbcA7t3 and pos: [ 170.    -489.623 -600.      -0.175]\n[-115.111  610.477 -726.222]\n[-117.577  610.477 -725.425]\n[-122.707  610.477 -723.767]\n[-130.501  610.477 -721.248]\n[-140.96   610.477 -717.867]\n[-154.083  610.477 -713.625]\n[-169.871  610.477 -708.522]\n[-188.216  610.477 -702.593]\n[-207.142  610.477 -696.475]\n[-224.092  610.477 -690.996]\n[-238.379  610.477 -686.379]\n[-250.001  610.477 -682.622]\n[-258.958  610.477 -679.727]\n[-265.251  610.477 -677.693]\n[-268.88   610.477 -676.52 ]\nWaypt w/ uuid: JfMsoTvyL8b5GRSnV4eufX and pos: [ 430.    -489.623 -700.       0.   ]\n[-269.845  610.477 -676.21 ]\n[-268.178  610.477 -676.851]\n[-263.897  610.477 -678.497]\n[-257.003  610.477 -681.149]\n[-247.495  610.477 -684.806]\n[-235.374  610.477 -689.468]\n[-220.64   610.477 -695.135]\n[-203.292  610.477 -701.807]\n[-184.625  610.477 -708.987]\n[-165.958  610.477 -716.166]\n[-147.29   610.477 -723.346]\n[-128.623  610.477 -730.526]\n[-109.956  610.477 -737.705]\n[ -91.289  610.477 -744.885]\n[ -72.679  610.477 -752.043]\n[ -55.867  610.477 -758.509]\n[ -41.668  610.477 -763.97 ]\n[ -30.083  610.477 -768.426]\n[ -21.111  610.477 -771.876]\n[ -14.753  610.477 -774.322]\n[ -11.008  610.477 -775.762]\nWaypt w/ uuid: TSU3bpg5xJyfyCH4spWEU6 and pos: [ 550.    -489.623 -750.       0.175]\n[  -9.864  610.477 -776.202]\n[  -8.399  610.477 -776.813]\n[  -4.349  610.477 -778.501]\n[   2.286  610.477 -781.265]\n[  11.506  610.477 -785.107]\n[  23.31   610.477 -790.025]\n[  37.699  610.477 -796.02 ]\n[  54.506  610.477 -803.023]\n[  70.171  610.477 -809.55 ]\n[  83.251  610.477 -815.   ]\n[  93.746  610.477 -819.373]\n[ 101.657  610.477 -822.669]\n[ 106.983  610.477 -824.889]\n[ 109.724  610.477 -826.031]\n[ 110.13   610.477 -826.2  ]\n\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:517:errback\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:580:_startRunCallbacks\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:662:_runCallbacks\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:1514:gotResult\n--- <exception caught here> ---\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:1443:_inlineCallbacks\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/python/failure.py:500:throwExceptionIntoGenerator\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robotbehavior.py:365:move\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:1445:_inlineCallbacks\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robotbehavior.py:409:_move\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robot.py:216:move\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/motionctl.py:482:move\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/motionctl.py:511:_expose_path\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robotbehavior.py:947:validate_path\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/defaultRobotVolumes.py:95:check\n", 'traceback': 'Traceback (most recent call last):\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 1514, in gotResult\n    current_context.run(_inlineCallbacks, r, g, status)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 1504, in _inlineCallbacks\n    status.deferred.errback()\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 517, in errback\n    self._startRunCallbacks(fail)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 580, in _startRunCallbacks\n    self._runCallbacks()\n--- <exception caught here> ---\n  File "/home/aft/_harv-aft-py-packages/aft-core/src/aft_core/asm/AsyncStateMachine.py", line 985, in _process_curr_state\n    result = yield self._stateTransDefer\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 662, in _runCallbacks\n    current.result = callback(current.result, *args, **kw)\n  File "/home/aft/_harv-aft-py-packages/aft-core/src/aft_core/twisted/twistutils.py", line 179, in trap_cancel\n    failType = fail.trap(defer.CancelledError, DeferredListError)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/python/failure.py", line 450, in trap\n    self.raiseException()\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/python/failure.py", line 475, in raiseException\n    raise self.value.with_traceback(self.tb)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 167, in maybeDeferred\n    result = f(*args, **kw)\n  File "/home/aft/_harv-aft-py-packages/aft-picker/src/aft_picker/pickerbehaviorbase.py", line 306, in state_handle_abort\n    raise child.finished_result.data\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 662, in _runCallbacks\n    current.result = callback(current.result, *args, **kw)\n  File "/home/aft/_harv-aft-py-packages/aft-core/src/aft_core/twisted/twistutils.py", line 179, in trap_cancel\n    failType = fail.trap(defer.CancelledError, DeferredListError)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/python/failure.py", line 450, in trap\n    self.raiseException()\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/python/failure.py", line 475, in raiseException\n    raise self.value.with_traceback(self.tb)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 1443, in _inlineCallbacks\n    result = current_context.run(result.throwExceptionIntoGenerator, g)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/python/failure.py", line 500, in throwExceptionIntoGenerator\n    return g.throw(self.type, self.value, self.tb)\n  File "/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robotclient.py", line 437, in track\n    ret = yield self._track(*args, **kwargs)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 1443, in _inlineCallbacks\n    result = current_context.run(result.throwExceptionIntoGenerator, g)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/python/failure.py", line 500, in throwExceptionIntoGenerator\n    return g.throw(self.type, self.value, self.tb)\n  File "/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robotclient.py", line 443, in _track\n    ret = yield self._try_motion(self.move_rpc, *args, **kwargs)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 1443, in _inlineCallbacks\n    result = current_context.run(result.throwExceptionIntoGenerator, g)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/python/failure.py", line 500, in throwExceptionIntoGenerator\n    return g.throw(self.type, self.value, self.tb)\n  File "/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robotclient.py", line 375, in _try_motion\n    ret = yield self.cur_try_motion_d\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 1443, in _inlineCallbacks\n    result = current_context.run(result.throwExceptionIntoGenerator, g)\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/python/failure.py", line 500, in throwExceptionIntoGenerator\n    return g.throw(self.type, self.value, self.tb)\n  File "/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robotclient.py", line 384, in _try_motion_worker\n    ret = yield self.active_motion_rpc_d\n  File "/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py", line 662, in _runCallbacks\n    current.result = callback(current.result, *args, **kw)\n  File "/home/aft/_harv-aft-py-packages/aft-core/src/aft_core/zmq/ZmqClient.py", line 353, in _handle_resp\n    raise TwistZmqException(resp)\naft_core.tools.TwistZmqException.TwistZmqException: Twist ZMQ Error:\nInvokation: InvokationTuple(servName=\'robot.4\', cmd=\'move\', args=([(Pos: [ 325.    -489.623 -650.1      0.   ], Spd: 2000.000, RotSpd: 205.714, Tol: 20.000, sysid: None, uuid: D29LnWwBsdRm2cgfBYpTjz, ts: 1686775369.826138, simultaneous: True, strict: True), (Pos: [ 170.    -489.623 -600.     -10.   ], Spd: 1000.000, RotSpd: 102.857, Tol: 0.100, sysid: None, uuid: bxeU86R4mSyqNhmcbcA7t3, ts: 1686775370.1318908, simultaneous: True, strict: True), (Pos: [ 430.    -489.623 -700.       0.   ], Spd: 1000.000, RotSpd: 102.857, Tol: 0.100, sysid: None, uuid: JfMsoTvyL8b5GRSnV4eufX, ts: 1686775370.5533156, simultaneous: True, strict: True), (Pos: [ 550.    -489.623 -750.      10.   ], Spd: 1000.000, RotSpd: 102.857, Tol: 0.100, sysid: None, uuid: TSU3bpg5xJyfyCH4spWEU6, ts: 1686775370.8258698, simultaneous: True, strict: True)],), kwargs={\'clear\': True}, ts=1686775369.4172866)\nType: <class \'aft_tbot.exceptions.RangeOfMotionException.RangeOfMotionException\'>\nMessage: \ntray/grip_org violations: \nWaypt w/ uuid: D29LnWwBsdRm2cgfBYpTjz and pos: [ 325.    -489.623 -650.1      0.   ]\n[-118.15   657.558 -691.572]\n[-116.873  639.224 -705.096]\n[-115.909  625.39  -715.3  ]\n[-115.259  616.055 -722.186]\n[-114.922  611.22  -725.752]\nWaypt w/ uuid: bxeU86R4mSyqNhmcbcA7t3 and pos: [ 170.    -489.623 -600.      -0.175]\n[-115.111  610.477 -726.222]\n[-117.577  610.477 -725.425]\n[-122.707  610.477 -723.767]\n[-130.501  610.477 -721.248]\n[-140.96   610.477 -717.867]\n[-154.083  610.477 -713.625]\n[-169.871  610.477 -708.522]\n[-188.216  610.477 -702.593]\n[-207.142  610.477 -696.475]\n[-224.092  610.477 -690.996]\n[-238.379  610.477 -686.379]\n[-250.001  610.477 -682.622]\n[-258.958  610.477 -679.727]\n[-265.251  610.477 -677.693]\n[-268.88   610.477 -676.52 ]\nWaypt w/ uuid: JfMsoTvyL8b5GRSnV4eufX and pos: [ 430.    -489.623 -700.       0.   ]\n[-269.845  610.477 -676.21 ]\n[-268.178  610.477 -676.851]\n[-263.897  610.477 -678.497]\n[-257.003  610.477 -681.149]\n[-247.495  610.477 -684.806]\n[-235.374  610.477 -689.468]\n[-220.64   610.477 -695.135]\n[-203.292  610.477 -701.807]\n[-184.625  610.477 -708.987]\n[-165.958  610.477 -716.166]\n[-147.29   610.477 -723.346]\n[-128.623  610.477 -730.526]\n[-109.956  610.477 -737.705]\n[ -91.289  610.477 -744.885]\n[ -72.679  610.477 -752.043]\n[ -55.867  610.477 -758.509]\n[ -41.668  610.477 -763.97 ]\n[ -30.083  610.477 -768.426]\n[ -21.111  610.477 -771.876]\n[ -14.753  610.477 -774.322]\n[ -11.008  610.477 -775.762]\nWaypt w/ uuid: TSU3bpg5xJyfyCH4spWEU6 and pos: [ 550.    -489.623 -750.       0.175]\n[  -9.864  610.477 -776.202]\n[  -8.399  610.477 -776.813]\n[  -4.349  610.477 -778.501]\n[   2.286  610.477 -781.265]\n[  11.506  610.477 -785.107]\n[  23.31   610.477 -790.025]\n[  37.699  610.477 -796.02 ]\n[  54.506  610.477 -803.023]\n[  70.171  610.477 -809.55 ]\n[  83.251  610.477 -815.   ]\n[  93.746  610.477 -819.373]\n[ 101.657  610.477 -822.669]\n[ 106.983  610.477 -824.889]\n[ 109.724  610.477 -826.031]\n[ 110.13   610.477 -826.2  ]\n\nServer Side Traceback: Traceback: <class \'aft_tbot.exceptions.RangeOfMotionException.RangeOfMotionException\'>: \ntray/grip_org violations: \nWaypt w/ uuid: D29LnWwBsdRm2cgfBYpTjz and pos: [ 325.    -489.623 -650.1      0.   ]\n[-118.15   657.558 -691.572]\n[-116.873  639.224 -705.096]\n[-115.909  625.39  -715.3  ]\n[-115.259  616.055 -722.186]\n[-114.922  611.22  -725.752]\nWaypt w/ uuid: bxeU86R4mSyqNhmcbcA7t3 and pos: [ 170.    -489.623 -600.      -0.175]\n[-115.111  610.477 -726.222]\n[-117.577  610.477 -725.425]\n[-122.707  610.477 -723.767]\n[-130.501  610.477 -721.248]\n[-140.96   610.477 -717.867]\n[-154.083  610.477 -713.625]\n[-169.871  610.477 -708.522]\n[-188.216  610.477 -702.593]\n[-207.142  610.477 -696.475]\n[-224.092  610.477 -690.996]\n[-238.379  610.477 -686.379]\n[-250.001  610.477 -682.622]\n[-258.958  610.477 -679.727]\n[-265.251  610.477 -677.693]\n[-268.88   610.477 -676.52 ]\nWaypt w/ uuid: JfMsoTvyL8b5GRSnV4eufX and pos: [ 430.    -489.623 -700.       0.   ]\n[-269.845  610.477 -676.21 ]\n[-268.178  610.477 -676.851]\n[-263.897  610.477 -678.497]\n[-257.003  610.477 -681.149]\n[-247.495  610.477 -684.806]\n[-235.374  610.477 -689.468]\n[-220.64   610.477 -695.135]\n[-203.292  610.477 -701.807]\n[-184.625  610.477 -708.987]\n[-165.958  610.477 -716.166]\n[-147.29   610.477 -723.346]\n[-128.623  610.477 -730.526]\n[-109.956  610.477 -737.705]\n[ -91.289  610.477 -744.885]\n[ -72.679  610.477 -752.043]\n[ -55.867  610.477 -758.509]\n[ -41.668  610.477 -763.97 ]\n[ -30.083  610.477 -768.426]\n[ -21.111  610.477 -771.876]\n[ -14.753  610.477 -774.322]\n[ -11.008  610.477 -775.762]\nWaypt w/ uuid: TSU3bpg5xJyfyCH4spWEU6 and pos: [ 550.    -489.623 -750.       0.175]\n[  -9.864  610.477 -776.202]\n[  -8.399  610.477 -776.813]\n[  -4.349  610.477 -778.501]\n[   2.286  610.477 -781.265]\n[  11.506  610.477 -785.107]\n[  23.31   610.477 -790.025]\n[  37.699  610.477 -796.02 ]\n[  54.506  610.477 -803.023]\n[  70.171  610.477 -809.55 ]\n[  83.251  610.477 -815.   ]\n[  93.746  610.477 -819.373]\n[ 101.657  610.477 -822.669]\n[ 106.983  610.477 -824.889]\n[ 109.724  610.477 -826.031]\n[ 110.13   610.477 -826.2  ]\n\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:517:errback\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:580:_startRunCallbacks\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:662:_runCallbacks\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:1514:gotResult\n--- <exception caught here> ---\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:1443:_inlineCallbacks\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/python/failure.py:500:throwExceptionIntoGenerator\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robotbehavior.py:365:move\n/home/aft/master/venv/lib/python3.8/site-packages/twisted/internet/defer.py:1445:_inlineCallbacks\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robotbehavior.py:409:_move\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robot.py:216:move\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/motionctl.py:482:move\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/motionctl.py:511:_expose_path\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/robotbehavior.py:947:validate_path\n/home/aft/_harv-aft-py-packages/aft-tbot/src/aft_tbot/motion/defaultRobotVolumes.py:95:check\n\n', 'code': 14, 'aux_uuid': '069fa944-0af4-11ee-868f-3a5051573772', 'ep': 'picker.4'}[0m[27m
[36m[20230614T134249.526] [TRANSITION] [harvester.fsm] -- Exiting: state_run[0m[27m
[20230614T134249.526] [DEBUG] [harvester.fsm] -- Service Lockup check stopped
[20230614T134249.526] [INFO] [harvester.fsm] -- Starting Behavior Termination
[20230614T134249.527] [INFO] [harvester.fsm] -- Calling Behavior stopListening
[20230614T134249.529] [INFO] [harvester.fsm] -- Start Behavior Terminate - defer initialized
[20230614T134249.531] [DEBUG] [harvester.fsm] -- Collecting harvester Unhandled error info
[20230614T134249.531] [DEBUG] [HarvMonitor ] -- Generating master sysmon reports
[36m[20230614T134249.539] [TRANSITION] [harv.beh.fsm] -- Exiting: state_error[0m[27m
[20230614T134250.112] [DEBUG] [servact     ] -- Disable Batch: {'picker.2', 'furrow.0', 'picker.4', 'picker.1'}
[20230614T134250.223] [DEBUG] [servact     ] -- Disable Batch: {'gripper.4', 'tracker.4', 'tracker.2', 'gripper.1', 'tracker.1', 'gripper.2'}
[20230614T134251.534] [INFO] [servact     ] -- Service Disable Operation - Complete
